testcontainers-dapr/src/test/resources/components/statestore.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
Integration tests, please.

Reply:
check my previous answer, and let's catch up about this.